lua/nvim-treesitter/parsers.lua

```diff
@@ -528,9 +528,22 @@ list.hcl = {
 list.markdown = {
   install_info = {
     url = "https://github.com/MDeiml/tree-sitter-markdown",
+    location = "tree-sitter-markdown",
     files = { "src/parser.c", "src/scanner.cc" },
-    branch = "main",
+    branch = "split_parser",
+  },
+  maintainers = { "@MDeiml" },
+  experimental = true,
+}
+
+list.markdown_inline = {
+  install_info = {
+    url = "https://github.com/MDeiml/tree-sitter-markdown",
+    location = "tree-sitter-markdown-inline",
```

Comment:
```suggestion
    location = "tree-sitter-markdown/tree-sitter-markdown-inline",
```

Reply:
we wanted to trim the unnecessary prefix from `location` anyway. It's just confusing...

Reply:
@MDeiml tried to fix this here https://github.com/nvim-treesitter/nvim-treesitter/pull/3067